docs/upgrade/2.10.1_to_2.11.0.rst

```diff
@@ -0,0 +1,104 @@
+.. _latest_upgrade_guide:
+
+Upgrade from 2.10.1 to 2.11.0
+=============================
+
+Preparing for the Ubuntu 24.04 (Noble) migration
+------------------------------------------------
+
+The 2.11.0 release includes a number of features that will help ensure
+your SecureDrop server is prepared for the automated migration to Ubuntu 24.04
+(Noble) in early 2025.
+
+After updating to SecureDrop 2.11.0 using the steps below, it's important
+that you review our :doc:`Ubuntu 24.04 (Noble) migration guide <../admin/maintenance/noble_migration_prep>`,
+and run the ``securedrop-noble-migration-check`` script prior to January 31st, 2025
```

Comment:
Sorry for tweaking this again, now that we have the language in the pre-announcement post, can we reuse it here? (slightly tweaked to be post-release)

SecureDrop 2.11.0 will automatically run checks to ensure all servers are ready for migration to Ubuntu 24.04 (Noble). If issues are found, a banner will be displayed in the Journalist Interface to both admins and journalists. Administrators are encouraged to review the guide explaining how to resolve any errors and perform any necessary steps before Jan. 31st, 2025.

Reply:
Absolutely, good call!